lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -1921,7 +1921,8 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
     (void)ID->getDecls();
 
     auto target = ID->getModule();
-    if (!getASTContext().LangOpts.PackageName.empty() &&
+    if (target && // some module might be on a block list so need to check for nil
```

Comment:
The comment is a bit off, `target` is null when the import fails to load. The blocklist was related only because it lists modules with broken imports, this fails only on invalid code which were all blocklisted modules.